kolibri/core/assets/src/vue/progress-bar/index.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+
+  <div>
+    <div class="visuallyhidden" id="progress-bar-label">{{ $tr('progressComplete') }}</div>
```

Comment:
Or is it supposed to work on other types of content? I see it on PDF content viewer too...